RDKShell/RDKShell.cpp

```diff
@@ -995,6 +995,11 @@ namespace WPEFramework {
             {
                 gWillDestroyEventWaitTime = atoi(willDestroyWaitTimeValue); 
             }
+
+            m_timer.start(RECONNECTION_INITIAL_DELAY_MILLISECONDS);
```

Comment:
Can we make it to start immediately with timeout of 0

Reply:
Changed delay to 0.